src/60_Samples_%26_Templates/Hotel.html

```diff
@@ -17,279 +18,149 @@
+                        min="2017-10-26"
+                        month-format="MMM"
+                        class="pt1 ml1">
+          <input amp-date-placeholder-start placeholder="arriving" id="arriving"/>
+          <input amp-date-placeholder-end placeholder="leaving" id="leaving"/>
+      </amp-date-picker>
+      <button class="ampstart-btn caps m1">Check Availability</button>
+    </form>
+
+    <!-- ## Additional filter -->
+    <!--
+    We implemented a filter to show only pet friendly rooms by adding a checkbox and
+    setting a new variable, `filterPetFriendly`, to the value of the filter.
+    -->
+    <label class="ml1">Show only pet friendly rooms: 
+      <input type="checkbox" class="m1" on="change:AMP.setState({
+        showPetFriendlyOnly: event.checked
+      })">
+    </label>
+  <!-- ## Filter function -->
+  <!--
+    We use `amp-bind-macro` to implement a function which filter the `amp-list` content
+    based on the pet friendly selection filter. By using `amp-bind-macro` we avoid repeating
+    the filter expression multiple times across the page.
+  -->
+  <amp-bind-macro id="filteredRooms"
+    expression="rooms.filter(room => showPetFriendlyOnly ? room.petFriendly : true)">
+  </amp-bind-macro>
+
+  <!-- ## Initial State -->
+  <!--
+    We are using `amp-state` to set the initial value of the variable `rooms` to the list of rooms
+    returned by the server. 
+  -->
+  <amp-state id="rooms" src="https://amp-by-example-live.glitch.me/rooms"></amp-state>
+
+  <!-- ## Rooms list -->
+  <!--
+    We are using `amp-list` to show a list of rooms. We bind the value of the `src` variable
+    to the result of the function `filterRooms` so that we can show the resulting list after the user selection.
+
+    By clicking on an image, you can see an enlarged version: we are in fact using the `amp-lightbox-gallery`
+    by adding the `lightbox` parameter to every `amp-img` component; the `alt` parameter content adds also
+    a caption for every image in the lightbox gallery.
+
+    `amp-list` takes all the height specified by the `height` attribute and after filtering, it could take unused space:
+    we are resizing by binding the `height` attribute to the result of the height of the image plus the margins multiplied by
+    the length of the list.
+  -->
+  <amp-list class="rooms" 
+  layout="fixed-height" height="800"
+  src="https://amp-by-example-live.glitch.me/rooms" [src]="filteredRooms()" items="."
+  [height]="(80 + (16*2)) * filteredRooms().length">
     <template type="amp-mustache">
-      <div class="{{type}}">
-        <a href="{{url}}">{{attraction}}</a>
-      </div>
+    <amp-img width="108" height="80" src="{{img}}" lightbox alt="{{name}}-{{desc}}" class="m1"></amp-img>
+    <div class="m1">{{name}} {{#petFriendly}}&#128062;{{/petFriendly}}</div>
+    <button class="ampstart-btn caps m1 mb3">BOOK</button>
     </template>
   </amp-list>
-</div>
+    <!-- ## Rooms list Empty result -->
```

Comment:
please read about title case and how to correctly use it.

Reply:
done